src/sdk.js

```diff
@@ -20,6 +20,8 @@ Formio.use(OpenFormsModule);
 // use our own template library
 Templates.current = OFLibrary;
 
+const {REACT_APP_RECAPTCHA_SITE_KEY} = process.env;
```

Comment:
This means you need a key at _build_ time, which we don't have. This should be an embed option instead of getting from the env.